src/meta-srv/src/handler/response_header_handler.rs

```diff
@@ -28,18 +28,15 @@ impl HeartbeatHandler for ResponseHeaderHandler {
 
     async fn handle(
         &self,
-        req: &HeartbeatRequest,
+        _req: &HeartbeatRequest,
```

Comment:
`_req` is not used, better remove it

Reply:
This parameter is used by other implemetations of `HeartbeatHandler`, like here:

https://github.com/GreptimeTeam/greptimedb/blob/765d1277ee6b71b2918e79cc478b73e4ca5388df/src/meta-srv/src/handler/collect_cluster_info_handler.rs#L105-L114